src/instantx.h

```diff
@@ -182,6 +197,9 @@ class CTxLockRequest
     CAmount GetMinFee() const;
     int GetMaxSignatures() const;
 
+    // checks if related transaction is "simple" to lock it automatically
```

Comment:
Shouldn't the `//` be `///` ?

If not, what is the difference in usecase

Reply:
Yes,  you are right,  it should be `///` really. But this class hasn't comments for other function, so Doxygen docs generation is not working for this class right now,  that's why I didn't add full comments to new functions in this PR. I'll add them in additional commit if someone will request some changes for this PR.

Reply:
A separate PR for documenting these classes would probably be cleaner (and something I have in mind for @PastaPastaPasta in the future if he's up for it 😉).